File: BonjourApi/BonjourApi/Models/NearbyUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BonjourApi.Models
{
    public class NearbyUser
    {
        public int Id { get; set; }
        public Boolean isFriend { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public int friendshipId { get; set; }
        public double latitude { get; set; }
        public double longitude { get; set; }
    }
}

File: android-app/bonjour2/app/src/main/java/com/example/bonjour/NearbyUser.java
package com.example.bonjour;

/**
 * Created by FATEHULLAH on 4/6/2015.
 */
public class NearbyUser
{
    public int Id;
    public Boolean isFriend;
    public String Name;
    public String Email;
    public int friendshipId;
    public double latitude;
    public double longitude;


    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public Boolean getIsFriend() {
        return isFriend;
    }

    public void setIsFriend(Boolean isFriend) {
        this.isFriend = isFriend;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getEmail() {
        return Email;
    }

    public void setEmail(String email) {
        Email = email;
    }

    public int getFriendshipId() {
        return friendshipId;
    }

    public void setFriendshipId(int friendshipId) {
        this.friendshipId = friendshipId;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }
}